### DISCUSSION TITLE
image_processor_config

### DISCUSSION THREAD

**Discussion by kghezelbash** [hezarai/hezar#167] [General]
Hi again, =)
Have you removed image_processor_config field from OCRDatasetConfig initialization recently? where should we indicate the OCRDatasetConfig in the training process now?

**Comment by arxyzan** [comment]
@kghezelbash Hi what's up!
Yes, in the latest release which is described in https://github.com/hezarai/hezar/releases/tag/0.39.0 , all datasets now get a `preprocessor` in their constructor. This way there is no need to provide preprocessor configs in the dataset's configs. You can define any preprocessor you like and pass it to the dataset.

See the new training tutorials we added recently upon this release at https://hezarai.github.io/hezar/tutorial/training/license_plate_recognition.html

Let me know if you have any more questions or challenges.

**Reply by kghezelbash** [reply]
Thank you. =)

**Reply by arxyzan** [reply]
You're welcome. Is the issue resolved?
